tests/foreman/ui/test_activationkey.py

```diff
@@ -97,9 +98,9 @@ def test_positive_end_to_end_register(session, repos_collection, rhel7_contentho
 
     :CaseImportance: High
     """
-    org = entities.Organization().create()
+    org = function_entitlement_manifest_org
     lce = entities.LifecycleEnvironment(organization=org).create()
-    repos_collection.setup_content(org.id, lce.id, upload_manifest=True)
+    repos_collection.setup_content(org.id, lce.id, upload_manifest=False)
```

Comment:
`upload_manifest=False` is not required as its by_default false. Remove it !

Optional!

Also @shweta83 You need to refactor `setup_content` function thats using manifest helper module for uploading the manifest while doing the manifester refactor with helpers !